### PR TITLE
allow disabling final color clamping

### PR DESCRIPTION
**Pull request by RaananW**
At the moment there is no way of disabling the final color clamp when using the PBR material (clamp to 30).
This sets clamping as the default value, but allows the dev to disable this using the image processing configuration.

### CODE REVIEW

File: src/Materials/Background/backgroundMaterial.ts [hunk 1]
```diff
@@ -123,6 +123,7 @@ class BackgroundMaterialDefines extends MaterialDefines implements IImageProcess
     public SAMPLER3DGREENDEPTH = false;
     public SAMPLER3DBGRMAP = false;
     public IMAGEPROCESSINGPOSTPROCESS = false;
+    public CLAMPFINALCOLOR = true;
```
Comment (sebavan) [inline src/Materials/Background/backgroundMaterial.ts:126]: As true is not the default value in rebuild, I think you either need to rename to a negative name like UNCLAMPFINALCOLOR = false, or override the reset to force the value back to true in all the defines objects IIRC.

**Review by sebavan — COMMENTED** [review]
Also this should be checked with the NodeMaterial as I can not remember how is it applied there ?

File: src/Materials/Background/backgroundMaterial.ts (hunk 1 above)
Comment (RaananW) [reply src/Materials/Background/backgroundMaterial.ts:126]: UNCLAMPFINALCOLOR sounded wrong...
But I agree that default false would be better.

Let's see what I can do

Comment (RaananW) [reply src/Materials/Background/backgroundMaterial.ts]: CLAMPFINALCOLOR => SKIPFINALCOLORCLAMP

**Comment by RaananW** [comment]
> Also this should be checked with the NodeMaterial as I can not remember how is it applied there ?

It is applied to the node material (has to be because it extends the image processing interface). Not sure how to use it there though.

**Review by sebavan — APPROVED** [review]

**Review by Popov72 — APPROVED** [review]

**Comment by Popov72** [comment]
The `ImageProcessingBlock` does not clamp the value so the changes won't have any impact on it.

**Pull request merged**